packages/contracts/test/utils/chai_setup.ts

```diff
@@ -2,12 +2,20 @@ import * as chai from 'chai';
 import chaiAsPromised = require('chai-as-promised');
 import ChaiBigNumber = require('chai-bignumber');
 import * as dirtyChai from 'dirty-chai';
+import { BigNumber } from 'bignumber.js';
 
 export const chaiSetup = {
     configure() {
         chai.config.includeStack = true;
         chai.use(ChaiBigNumber());
         chai.use(dirtyChai);
         chai.use(chaiAsPromised);
+        
+        // Node uses '.inspect()' instead of '.toString()' for log messages
+        // HACK: Typescript won't allow me to mess with BigNumber.prototype
+        // directly, so I create an instance and then get the prototype.
+        Object.getPrototypeOf(new BigNumber(0)).inspect = function() {
```

Comment:
This is should be done properly.

Reply:
@Recmo I'd also be for moving this into our configured exported `BigNumber` in `@0xproject/utils`. What do you think @LogvinovLeon?